fpmsyncd/fpmlink.cpp

```diff
@@ -120,12 +200,28 @@ uint64_t FpmLink::readData()
 
         if (hdr->msg_type == FPM_MSG_TYPE_NETLINK)
         {
-            nl_msg *msg = nlmsg_convert((nlmsghdr *)fpm_msg_data(hdr));
+            bool isRaw = false;
+
+            nlmsghdr *nl_hdr = (nlmsghdr *)fpm_msg_data(hdr);
+
+            isRaw = isRawProcessing(nl_hdr);
```

Comment:
Can you add some comment here on what is Raw processing?

Reply:
Add as suggested